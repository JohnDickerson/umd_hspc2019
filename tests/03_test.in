9

5
11
22
25
31
55
77
101
1001
