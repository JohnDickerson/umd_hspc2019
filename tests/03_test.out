101
961
6421
10761
20381
112501
307421
692101
669171001
